### DISCUSSION TITLE
Lesson 9 : smartcontractkit/chainlink-brownie-contracts download failing

### DISCUSSION THREAD

**Discussion by kinjun751** [Cyfrin/foundry-full-course-cu#2405] [General]
I have difficulty in downloading the smartcontractkit/chainlink-brownie-contracts...
it gives out this error

'''
Error:
git submodule exited with code 128

'''

How will I work around this?

**Comment by EngrPips** [comment]
Hello @kinjun751, This is an internet connection problem. Please retry whenever your internet connection is more stable.
